### PR TITLE
Add Cayman theme and initial content, no styling

### PR DESCRIPTION
**Pull request by rightsaidjames**
Added theme and initial content, no styling yet.

Here's what it looks like:

### CODE REVIEW

File: _site/images/bla.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+To be deleted
```
Comment (will-banked) [inline _site/images/bla.txt]: What's this for?

File: _layouts/default.html [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    <link rel="preload" href="https://fonts.googleapis.com/css?family=Open+Sans:400,700&display=swap" as="style" type="text/css" crossorigin>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta name="theme-color" content="#157878">
+    <link rel="icon" href="/_includes/favicon.png">
+    <meta name="apple-mobile-web-app-status-bar-style" content="black-translucent">
+    <link rel="stylesheet" href="{{ '/assets/css/style.css?v=' | append: site.github.build_revision | relative_url }}">
+    {% include head-custom.html %}
+  </head>
+  <body>
+    <a id="skip-to-content" href="#content">Skip to the content.</a>
```
Comment (will-banked) [inline _layouts/default.html:17]: Screenshot 2023-04-13 at 15 46 35
Is this link the button in the image?

Comment (will-banked) [reply _layouts/default.html:17]: Ah nvm realise this whole thing is the header basically.

Comment (rightsaidjames) [reply _layouts/default.html:17]: Yeah, I've hardcoded the button and the logo in the header, the rest of it comes from the `_config.yml` file or is just a core part of the theme.

File: _site/images/bla.txt (hunk 1 above)
Comment (rightsaidjames) [reply _site/images/bla.txt]: I thought I'd deleted it, will try to get rid of it again!

File: _layouts/default.html [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+    <header class="page-header" role="banner">
+      <h1 class="project-name">{{ default: site.title | default: site.github.repository_name }}</h1>
```
Comment (rightsaidjames) [inline _layouts/default.html:20]: I've removed the `page.title` variable from this element so that it always displays the site title.

**Review by will-banked — APPROVED** [review]
Consider this approved!

**Pull request merged**